packages/admin-ui/src/pages/packages/components/Info/ContainerList.tsx

```diff
@@ -65,7 +85,11 @@ export const ContainerList = ({ dnp }: { dnp: InstalledPackageData }) => {
           </span>
 
           {allContainersRunning ? (
-            <MdPauseCircleOutline onClick={() => onStartStop()} />
+            <MdPauseCircleOutline
+              onClick={() =>
+                isWifiPackage() ? onStartStopConfirm() : onStartStop()
```

Comment:
Checking if is Wifi package here won't cover clicking the other start stop button. You could check after this line if it's the wifi package 
https://github.com/dappnode/DNP_DAPPMANAGER/blob/3b4db52977783bc98cf7f7c01ae85252888834e0/packages/admin-ui/src/pages/packages/components/Info/ContainerList.tsx#L29
and conditionally show the dialog